ledfx/mood_detector.py

```
@@ -0,0 +1,953 @@
+            classify_sub_genre,
+            create_librosa_extractor,
+            is_librosa_available,
+        )
+    LIBROSA_MODULE_AVAILABLE = True
+except ImportError:
+    LIBROSA_MODULE_AVAILABLE = False
+
+    def is_librosa_available():
+        return False
+
+    def create_librosa_extractor(*args, **kwargs):
```

Ruff (0.14.8)

Unused function argument: `args`

(ARG001)